src/audio/CMakeLists.txt

```diff
@@ -128,7 +129,7 @@ endif()
 add_local_sources(sof
 	component.c
 	data_blob.c
-	buffer.c
+	buffers/comp_buffer.c
```

Comment:
We can do this in the next PR if needed (as CI is good), but does it make sense to rename like other similar files to legacy-buffer.c ?